### ISSUE TITLE
flowcb/log prints downloaded message even when download is off

### ISSUE THREAD

**Issue by mshak2** [MetPX/sarracenia#976]

**Comment by andreleblanc11** [comment]
Is there only a message in the log? Or does a download attempt also take place?

**Comment by andreleblanc11** [comment]
What does the log output look like?

**Comment by mshak2** [comment]
> Is there only a message in the log? Or does a download attempt also take place?

Yes it's only a message in the log. No downloading occurs but it is confusing to end users and Peter (and me).

**Comment by mshak2** [comment]
Steps to reproduce:
```
sr3 add subscribe/dd_amis
sr3 edit subscribe/dd_amis
# add a line: download off
sr3 foreground subscribe/dd_amis
```
When the foreground runs, you'll see downloaded messages, but nothing is downloaded.

**Comment by petersilva** [comment]
fix: 46c9c72f9ab9dca6022a66f716386e541ca0287f
